runtime/account_test.go

```diff
@@ -891,6 +884,9 @@ func getAccountKeyTestRuntimeInterface(storage *testAccountKeyStorage) *testRunt
 		decodeArgument: func(b []byte, t cadence.Type) (value cadence.Value, err error) {
 			return json.Decode(b)
 		},
+		validatePublicKey: func(publicKey *PublicKey) (bool, error) {
+			return true, nil
+		},
```

Comment:
I believe this is necessary because before, tests would set `isValid` to `true` to bypass validation checking or let it be `false` if it didn't matter for the test. Now that invalid public keys panic, those tests fail. But I'm not confident that this isn't masking some deeper problem.

Reply:
Maybe double check where `getAccountKeyTestRuntimeInterface` is/was called and see if it expected the key to be valid/invalid. See below

Reply:
Aha! The test runtime defaults PublicKey validation to `false` if the `validatePublicKey` field isn't specified. Most of the tests don't care if it's valid or not.

I'll make it default to `true`.